pkg/package.json

```diff
@@ -0,0 +1,24 @@
+{
```

Comment:
I'm not familiar with the `pkg` directory, is this for a readme and package.json that get pushed to npm?

Reply:
sorry, not sure what your question is

Reply:
oh, now I see :P ahhh, idk. looks that way to me

Reply:
maybe that should be gitignored?

Reply:
there we go, all fixed up (I think)

Reply:
sorry haha wasn't sure either 😬